Calibration/HcalAlCaRecoProducers/python/ALCARECOHcalCalIsolatedBunchFilter_Output_cff.py

```diff
@@ -0,0 +1,22 @@
+import FWCore.ParameterSet.Config as cms
+
+# output block for alcastream HCAL IsolatedBunch
+# output module 
+#  module alcastreamHcalIsolatedBunchOutput = PoolOutputModule
+
+OutALCARECOHcalCalIsolatedBunchFilter_noDrop = cms.PSet(
+    # use this in case of filter available
+    SelectEvents = cms.untracked.PSet(
+        SelectEvents = cms.vstring('pathALCARECOHcalCalIsolatedBunchFilter')
+    ),
+    outputCommands = cms.untracked.vstring( 
+        'keep *_hbhereco_*_*',
+        'keep *_ecalRecHit_*_*',
+        'keep *_hltTriggerSummaryAOD_*_*',
+        'keep *_TriggerResults_*_*',
+        )
```

Comment:
The new path is ALCARECOHcalCalIsolatedBunchSelector_cff.py which has the right parts selected. I shall make a new tag for the Filter (which is the offline version and does not avoid the prescale at HLT level for Jet triggers). I have not made the porting of this (nor the other to 8_0_X). I shall do that once it gets integrated to 8_1_X. Thanks Sunanda
________________________________
From: Marco Musich [notifications@github.com]
Sent: 26 July 2016 09:26
To: cms-sw/cmssw
Cc: Sunanda Banerjee; Mention
Subject: Re: [cms-sw/cmssw] bsunanda:Run2-alca45 Add a filter which filters out isolated bunch with Jet triggers (#14541)


In Calibration/HcalAlCaRecoProducers/python/ALCARECOHcalCalIsolatedBunchFilter_Output_cff.py<https://github.com/cms-sw/cmssw/pull/14541#discussion_r72200268>:

> +
> +# output block for alcastream HCAL IsolatedBunch
> +# output module
> +#  module alcastreamHcalIsolatedBunchOutput = PoolOutputModule
> +
> +OutALCARECOHcalCalIsolatedBunchFilter_noDrop = cms.PSet(
> +    # use this in case of filter available
> +    SelectEvents = cms.untracked.PSet(
> +        SelectEvents = cms.vstring('pathALCARECOHcalCalIsolatedBunchFilter')
> +    ),
> +    outputCommands = cms.untracked.vstring(
> +        'keep *_hbhereco_*_*',
> +        'keep *_ecalRecHit_*_*',
> +        'keep *_hltTriggerSummaryAOD_*_*',
> +        'keep *_TriggerResults_*_*',
> +        )


@bsunanda<https://github.com/bsunanda> please explicit the types of the products.
Also, there was mention in presentation<https://indico.cern.ch/event/558794/contributions/2254652/attachments/1315042/1969666/Phase1_13.pdf> that RAW products would have been kept, but I don't see them. Please clarify

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://github.com/cms-sw/cmssw/pull/14541/files/c6aa22d9a6bc4eea90d96557574f64c5b3ac3b54#r72200268>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AEzMum_7cNm2oxkTADaNHBIdDoM_mI8kks5qZbaogaJpZM4IguX7>.